lib/errorHandlers/suppressError.ts

```diff
@@ -59,7 +77,10 @@ function createPlatformVersionError(err, subCategory) {
   uiLine();
 }
 
-function shouldSuppressError(err, context = {}) {
+export function shouldSuppressError(
+  err: unknown,
+  context?: typeof ApiErrorContext
```

Comment:
Don't need typeof here

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1251/commits/146dcbafeabd6d865cb305cb61cf867207c31fbf